plugin/cassandra/plugin.go

```diff
@@ -90,106 +90,151 @@ package main
 				Title:    "Cassandra Host",
 				Help:     "The hostname or IP address of your Cassandra server.",
 				Default:  "127.0.0.1",
-				Required: true,
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
 				Name:     "port",
 				Type:     "port",
 				Title:    "Cassandra Port",
 				Help:     "The 'native transport' TCP port that Cassandra server is bound to, listening for incoming connections.",
 				Default:  "9042",
-				Required: true,
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
 				Name:     "user",
 				Type:     "string",
 				Title:    "Cassandra Username",
 				Help:     "Username to authenticate to Cassandra as.",
 				Default:  "cassandra",
-				Required: true,
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
-				Name:     "password",
+				Name:     "cassandra",
 				Type:     "password",
 				Title:    "Cassandra Password",
-				Help:     "Password to authenticate to Cassandra as.",
+				Help:     "Password to auth_userenv",
 				Default:  "cassandra",
-				Required: true,
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
-				Name:     "keyspace",
-				Type:     "string",
-				Title:    "Keyspace to Backup",
-				Help:     "The name of the keyspace to backup.",
-				Example:  "system",
-				Required: true,
+				Name:     "include_keyspaces",
+				Type:     "array",
+				Title:    "Keyspaces to Include in the Backup",
+				Help:     "The name of the keyspace to include in the backup.",
+				Example:  "['system']",
+				Required: false,
+			},
+			{
+				Mode:     "target",
+				Name:     "exclude_keyspaces",
+				Type:     "array",
+				Title:    "Keyspaces to Exclude from Backup",
+				Help:     "The name of the keyspace to exclude from backup.",
+				Example:  "[system']",
+				Default:  "['system_schema', 'system_distributed', 'system_auth', 'system', 'system_traces']",
+				Required: false,
+			},
+			{
+				Mode:     "target",
+				Name:     "save_users",
+				Type:     "boolean",
+				Title:    "to restore users and permissions along with the keyspaces that are restored",
+				Help:     "The name of the keyspace to exclude from backup.",
+				Example:  "true or false",
+				Required: false,
+			},
+			{
+				Mode:     "target",
+				Name:     "bindir",
+				Type:     "abspath",
+				Title:    "Path to binaries nodetool, sstableloader and cqlsh needed for the backup",
+				Help:     "where  to find binaries nodetool, sstableloader and cqlsh",
+				Example:  "true or false",
+				Default:  "/var/vcap/jobs/cassandra/bin",
+				Required: false,
 			},
-			plugin.Field{
+			{
 				Mode:     "target",
 				Name:     "datadir",
 				Type:     "abspath",
 				Title:    "Path to Cassandra data/ directory",
 				Help:     "The absolute path to the data/ directory that contains the Cassandra database files.",
 				Default:  "/var/vcap/store/cassandra/data",
-				Required: true,
+				Required: false,
```

Comment:
If this is no longer required, we should remove the default.  If the default is standing in for "no value", then it should still be considered _required_.